material-ui/package.json

```
@@ -0,0 +1,34 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "build": "remix build",
+    "dev": "remix dev",
+    "start": "remix-serve build"
+  },
+  "dependencies": {
+    "@emotion/cache": "*",
+    "@emotion/react": "*",
+    "@emotion/server": "*",
+    "@emotion/styled": "*",
+    "@fontsource/roboto": "*",
+    "@mui/icons-material": "^5.10.6",
+    "@mui/material": "*",
```

Let's use specific versions for non-Remix packages

```suggestion
    "@emotion/cache": "^11.10.3",
    "@emotion/react": "^11.10.4",
    "@emotion/server": "^11.10.0",
    "@emotion/styled": "^11.10.4",
    "@fontsource/roboto": "^4.5.8",
    "@mui/icons-material": "^5.10.6",
    "@mui/material": "^5.10.6",
```